sdk/eventhub/event-hubs/src/eventHubConsumerClientModels.ts

```diff
@@ -155,6 +143,10 @@ export interface SubscribeOptions extends TracingOptions {
    * passing the data to user code for processing. If not provided, it defaults to 60 seconds.
    */
   maxWaitTimeInSeconds?: number;
+  /**
+   * The starting partition position used if no checkpoint is found.
```

Comment:
```suggestion
   * The event position in a partition to start receiving events from if no checkpoint is found.
   * Pass a map of partition id to position if you would like to use different starting
   * position for each partition.
```

Reply:
Fixed with https://github.com/Azure/azure-sdk-for-js/pull/6632/commits/42bee48bc5317a9c01a5811f60bb6aeb7d48032d